airflow-core/src/airflow/ui/public/i18n/locales/he/dags.json

```diff
@@ -19,7 +19,8 @@
       "active": "פעיל",
       "all": "הכל",
       "paused": "מושהה"
-    }
+    },
+    "runIdPatternFilter": "חפש הרצת Dag"
```

Comment:
I thought DagRun is a terminology we don't translate?

Reply:
It's pretty much clear IMO - so I wouldn't change it anyway